openfoodfacts/ocr.py

```diff
@@ -1130,6 +1139,43 @@ def __init__(self, data: JSONType):
         self.description = data["description"]
 
 
+class FaceAnnotation:
+    __slots__ = (
+        "detection_confidence",
+        "joy_likelihood",
+        "sorrow_likelihood",
+        "anger_likelihood",
+        "surprise_likelihood",
+        "under_exposed_likelihood",
+        "blurred_likelihood",
+        "headwear_likelihood",
+    )
+
+    def __init__(self, data: JSONType):
+        self.detection_confidence = data.get("detectionConfidence", 0.0)
+        self.joy_likelihood = SafeSearchAnnotationLikelihood[
```

Comment:
I think we shouldn't use `SafeSearchAnnotationLikelihood`, as this is not related to SafeSearchAnnotation. Google Cloud has a single `Likelihood` class (https://cloud.google.com/python/docs/reference/vision/latest/google.cloud.vision_v1.types.Likelihood), Can you create a new `Likelihood` enum with the current content of `SafeSearchAnnotationLikelihood`, and make `SafeSearchAnnotationLikelihood` inherit from it?
So that we keep the SDK compatibility for those who rely on SafeSearchAnnotationLikelihood. We can drop `SafeSearchAnnotationLikelihood` in favor of `Likelihood` in a future version.